### PR TITLE
adding ability to listen for compose close events

### PR DESCRIPTION
**Pull request by yellowandy**

### CODE REVIEW

**Comment by KartikTalwar** [comment]
Sorry for getting to this late, would you mind updating the readme?

File: src/gmail.js [hunk 1]
```diff
@@ -1274,7 +1283,12 @@ var Gmail = function(localJQuery) {
       return true;
 
     // support for gmail interface load event
-    } else if(action == 'load') {
+    }
+    else if(action == 'compose_cancelled') {
+        console.log('set compose cancelled callback');
```
Comment (yellow-beard) [inline src/gmail.js]: might want to remove this console log?

**Comment by irtzaqc** [comment]
ETA on merge? I kinda need this functionality

**Comment by josteink** [comment]
This looks like useful functionality, and I'll see if I can get it merged properly.

**Comment by josteink** [comment]
Due the several factorsthis PR was almost quite impossible to get merged cleanly:

* time since it was published
* multiple merge-bubbles
* mixing of different issues in the same PR

In the end I settled for a git cherry-pick to get the compose_cancelled functionality merged and left everything else out. It took me almost an hour :(

Hopefully this should settle it for everyone.

**Pull request closed without merging**